tests/test_app.py

```
@@ -7,3 +7,5 @@ def test_greet(self):
 
 if __name__ == "__main__":
     unittest.main()
+
+return true;
```

Ruff (0.12.2)

`return` statement outside of a function/method

(F706)